iOSDeviceManager/Commands/AppInfoCommand.m

```diff
@@ -30,7 +30,7 @@ + (NSString *)name {
         [options addObject:[CommandOption withShortFlag:DEVICE_ID_FLAG
                                                longFlag:@"--device-id"
                                              optionName:DEVICE_ID_OPTION_NAME
-                                                   info:@"iOS Simulator GUID or 40-digit physical device ID"
+                                                   info:@"iOS Simulator GUID or 40-digit physical device ID or alias"
```

Comment:
Let's make this more succinct:

"Simulator GUID, device UDID, or an alias"

Reply:
Changed for all command that use -d parameter